policies/Sponsorship_Policy.md

```diff
@@ -0,0 +1,153 @@
+     * Value of sponsorship in $US and local currency (where applicable);
+     * Details of what is being supplied under the sponsorship e.g. financial aid, 
+resources etc;
+     * Any conditions that apply to the sponsorship;
+     * Whether permission to display logos, company details and links to their 
+websites has been provided;
+     * Contact details for the Organisation, Individual or Entity.
+
+* Acknowledgement of Sponsors on OpenSSL's website, including logos, company 
+details and links to external websites will only ocurr with express permission 
+of the sponsoring organisation, individual or entity.  At the end of the 
+sponsorship period, where sponsorship is not renewed, acknowledgements will be 
+removed from the OpenSSL website.
+* For sponsors on [GitHub Sponsors] that have elected to be _public_, it is 
+deemed that they have provided express permission for such acknowledgement as 
+provided by [GitHub Sponsors] and to receiving direct contact from OpenSSL.
+
+
+#### OpenSSL Providing Sponsorship
+
+
+Below sets out how OpenSSL will assess requests to provide sponsorship.
+
+The OMC will assess each application for sponsorship, under the following 
+criteria:
+* Does sponsoring support OpenSSL's Mission and values, it's goals, objectives 
+and priorities;
+* Could sponsoring be perceived as a conflict of interest, or impartial;
+* Does sponsoring breach any legal or regulatory requirements;
+* Will sponsoring damage OpenSSL's reputation;
+* Could sponsoring cause offense or distress to OpenSSL's community;
+* What is the financial implication of sponsoring to OpenSSL;
+* What are the benefits of sponsoring this Organisation, Individual or Entity;
+* Does sponsoring further Open Source principles.
+
+Where a sponsorship is reported to the OMC as potentially inappropriate the 
+above criteria will be used to assess that sponsorship.
+
+**Recording Sponsorship provided by OpenSSL**
+All sponsorship arrangements with Organisations, Individuals or Entities that 
+OpenSSL sponsors must be documented and recorded in an appropriate system so 
+that it can be identified and managed.
+
+Details to be recorded in this system are:
+* Name of Organisation, Individual or Entity being sponsored;
+* Duration of sponsorship arrangement, including start and end dates;
+* Value of sponsorship in $US and local currency (where applicable);
+* Details of what is being supplied under the sponsorship e.g. financial aid, 
+resources etc;
+* Any conditions that apply to the sponsorship;
+* Contact details for the Organisation, Individual or Entity;
+* Reason for approving sponsorship.
+
+
+
+### Inappropriate Sponsorship
+
+The following entities have been deemed inappropriate sponsorship by the OMC.
+OpenSSL will not accept sponsorship from, nor provide sponsorship to, any of 
+the following:
+* Political Parties;
+* Governments;
+* Known criminal organisations or individuals.
```

Comment:
While the first two of these probably have pretty straightforward ways to determine whether they apply, there might be some variation of interpretation for this last one.  Do you have a more specific sense for how this would be determined (perhaps, in the OMC's assessment)?  I suspect that, e.g., the union of all governments' application of this label might end up being a surprisingly broad net.

Reply:
It is a tricky one as there's no definitive list.  Ultimately the OMC can reject any sponsorship if they deem it inappropriate so this is probably more a statement of intent. But will check with the OMC what they want to do.

Reply:
Tim has suggested removing the "Known criminal..." reference all together as rejecting that sponsorship would be possibly under the policy anyway, so no need to specifically list them.  So I will remove that line.